avocado/core/job.py

```diff
@@ -336,8 +335,8 @@ def soft_abort(msg):
                 os.unlink(proc_latest)
 
     def _start_sysinfo(self):
-        if hasattr(self.args, 'sysinfo'):
-            if self.args.sysinfo == 'on':
+        if 'sysinfo' in self.config:
+            if self.config.get('sysinfo') == 'on':
```

Comment:
I would combine those ifs in one ```if self.config.get('sysinfo', None) == 'on'```

Reply:
Yep, I tried to do a more straightforward conversion to ease review, but I was bothered by this also.  I'll add a commit changing the style all over on top of those.

Reply:
Actually, I thought there were many more like this, but I only found one.  So, I'm putting them on the same (first) commit.